dokku-vm/azuredeploy.json

```diff
@@ -69,17 +100,16 @@
         },
         "allowedValues": [
             "East US",
-            "East US 2",
-            "East Asia",
-            "West US",
```

Comment:
Is that incorrect? - I literally posted the output of my API responses verbatim.